packages/angular/projects/angular-sdk/README.md

```diff
@@ -0,0 +1,258 @@
+  <a href="https://github.com/open-feature/js-sdk/releases/tag/angular-sdk-v0.0.0">
+    <img alt="Release" src="https://img.shields.io/static/v1?label=release&message=v0.0.0&color=blue&style=for-the-badge" />
+  </a>
+  <!-- x-release-please-end -->
+  <br/>
+  <a href="https://codecov.io/gh/open-feature/js-sdk">
+    <img alt="codecov" src="https://codecov.io/gh/open-feature/js-sdk/branch/main/graph/badge.svg?token=3DC5XOEHMY" />
+  </a>
+  <a href="https://www.npmjs.com/package/@openfeature/angular-sdk">
+    <img alt="NPM Download" src="https://img.shields.io/npm/dm/%40openfeature%2Fangular-sdk" />
+  </a>
+</p>
+<!-- x-hide-in-docs-start -->
+
+[OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API
+for feature flagging that works with your favorite feature flag management tool or in-house solution.
+
+<!-- x-hide-in-docs-end -->
+
+## Overview
+
+The OpenFeature Angular SDK adds Angular-specific functionality to
+the [OpenFeature Web SDK](https://openfeature.dev/docs/reference/technologies/client/web).
+
+## Table of Contents
+
+- [Quick start](#quick-start)
+    - [Requirements](#requirements)
+    - [Install](#install)
+        - [npm](#npm)
+        - [yarn](#yarn)
+        - [Required peer dependencies](#required-peer-dependencies)
+    - [Minimal Example](#minimal-example)
+    - [Flag Types](#flag-types)
+        - [Boolean Feature Flag](#boolean-feature-flag)
+        - [Number Feature Flag](#number-feature-flag)
+        - [String Feature Flag](#string-feature-flag)
+        - [Object Feature Flag](#object-feature-flag)
+- [Inputs](#inputs)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Quick start
+
+### Requirements
+
+- ES2015-compatible web browser (Chrome, Edge, Firefox, etc)
+- Angular version 16+
+
+### Install
+
+#### npm
+
+```sh
+npm install --save @openfeature/angular-sdk
+```
+
+#### yarn
+
+```sh
+# yarn requires manual installation of the peer dependencies (see below)
+yarn add @openfeature/angular-sdk @openfeature/web-sdk @openfeature/core
+```
+
+#### Required peer dependencies
+
+The following list contains the peer dependencies of `@openfeature/angular-sdk`.
+See the [package.json](./package.json) for the required versions.
+
+* `@openfeature/web-sdk`
+* `@angular/common`
+* `@angular/core`
+
+#### Usage
+
+#### Module
+
+To include the OpenFeature Angular directives in your application, you need to import the `OpenFeatureModule` and
+configure it using the `forRoot` method.
+
+```typescript
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { OpenFeatureModule } from '@openfeature/angular-sdk';
+
+@NgModule({
+  declarations: [
+    // Other components
+  ],
+  imports: [
+    CommonModule,
+    OpenFeatureModule.forRoot({
+      provider: yourFeatureProvider,
+      // domainBoundProviders are optional, mostly needed if more than one provider is needed
+      domainBoundProviders: {
```

Comment:
Question: Should we include how to set a new Provider outside of using `OpenFeatureModule.forRoot`? For example, if someone needs to set a provider after the app initializes?

Reply:
Not a bad thing to add but also not something I'd consider a blocker. I think most Angular people would "just get it".

Reply:
I will add a sentence to this and also explaining that a service can simply be taken from the GlobalApi.